misc/assets/inforequests.svg

```diff
@@ -0,0 +1,248 @@
+<svg version="1.1" viewBox="0 0 1000 1350" xmlns="http://www.w3.org/2000/svg">
+  <style>
+      text {
+        font-family: sans-serif;
+      }
+      .external{
+        stroke-dasharray: 3pt;
+      }
+      .caption {
+        font-size: 12pt;
+        text-anchor: middle;
+        dominant-baseline: central;
+      }
+      .note {
+        font-size: 9pt;
+        text-anchor: middle;
+        dominant-baseline: ideographic;
+      }
+      .left-rotation{
+        transform: rotate(-90deg);
+      }
+      .right-rotation{
```

Comment:
`.right-rotation` sa tyka iba `.note`, tak radsej `.note.right-rotation`, nech sa to nemiesa s inymi vecami.

Btw, naco hldy nazov "right-rotation" a nie len "right"? Analogicky pre "left".

Reply:
:ok: